kstests-in-cloud.sh

```diff
@@ -79,7 +81,7 @@ Options:
 
     --remote-user            remote user for deployment of provisioned runners by ansible;
                              for example for Fedora cloud images it is "fedora";
-                             for RHEL could images it is "cloud-user"
+                             for RHEL cloud images it is "cloud-user"
```

Comment:
This shouldn't be part of this commit. I know it is a nitpick but it creates not-useful noise in a history and similar.